ingredient/README.md

```diff
@@ -0,0 +1,28 @@
+# @food-doods/ingredient
+
+## Development
+
+1. Install and start postgres:
+
+```
+brew install postgresql
+brew services start postgres
+```
+
```

Comment:
Could we setup a docker container for this instead?

It will keep setup minimal later on for different parts of the team.